### PR TITLE
Fix lint errors in binaries

### PR DESCRIPTION
**Pull request by bwester**
Fixes a bunch of warnings raised by running "gometalinter" on all the
binary packages. Changes mostly eliminate dead code, check error
return values, and adjust variable names/cases.

### CODE REVIEW

File: bin/p2-bin2pod/main.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func main() {
 	}
 
 	manifestBuilder.SetLaunchables(map[string]pods.LaunchableStanza{
-		string(podId()): stanza,
+		string(podID()): stanza,
```
Comment (mpuncel) [inline bin/p2-bin2pod/main.go]: while you're at it, can we make this `podID().String()`

File: bin/p2-rctl/main.go [hunk 2]
```diff
@@ -34,57 +34,57 @@ import (
...
-	CMD_SCHEDUP  = "schedule-update"
+	cmdCreateText   = "create"
+	cmdDeleteText   = "delete"
+	cmdReplicasText = "set-replicas"
+	cmdListText     = "list"
+	cmdGetText      = "get"
+	cmdEnableText   = "enable"
+	cmdDisableText  = "disable"
+	cmdRollText     = "rolling-update"
+	cmdSchedupText  = "schedule-update"
```
Comment (petertseng) [inline bin/p2-rctl/main.go]: perhaps `SchedUp`, but I'm a bit unsure whether that's a material improvement

**Comment by mpuncel** [comment]
👍

**Comment by petertseng** [comment]
Seems OK, 👍

**Comment by rudle** [comment]
:+1:

File: bin/p2-bin2pod/main.go (hunk 1 above)
Comment (bwester) [inline bin/p2-bin2pod/main.go]: Sounds good; will do.

File: bin/p2-rctl/main.go (hunk 2 above)
Comment (bwester) [inline bin/p2-rctl/main.go]: Elsewhere, there is already a `cmdSchedup`, and this just provides the text of the command. Mostly I just care about removing CAPS and not having two variables with the same name that vary only in the casing pattern used.

**Comment by bwester** [comment]
@mpuncel: I expect this to conflict with #525. Are you close to committing that one?

**Comment by bwester** [comment]
Updated: use `podID.String()`, rebase onto latest master. Will commit after tests pass.

**Pull request merged**